include/toml/toml.h

```diff
@@ -140,12 +146,15 @@ class Value {
     const Value* find(size_t index) const;
     Value* find(size_t index);
     Value* push(const Value& v);
+    Value* push(Value && v);
 
     // ----------------------------------------------------------------------
     // Others
 
     // Writer.
-    void write(std::ostream*, const std::string& keyPrefix = std::string()) const;
+    std::string getIndent(int indent) const;
+
+    void write(std::ostream*, int indent = INDENT_DISABLED, const std::string& keyPrefix = std::string()) const;
```

Comment:
Please keep the original argument order. This broke my app...

`void write(std::ostream*, const std::string& keyPrefix = std::string(), int indent = INDENT_DISABLED) const;`